src/backend/base/langflow/components/agents/JsonAgent.py

```diff
@@ -25,7 +25,7 @@ def build_agent(self) -> AgentExecutor:
         path = Path(self.path)
         if self.path.endswith("yaml") or self.path.endswith("yml"):
             with path.open() as file:
-                yaml_dict = yaml.load(file, Loader=yaml.FullLoader)
+                yaml_dict = yaml.full_load(file)
```

Comment:
Is this data trusted ? If not, we should use `yaml.safe_load()` here. Is there a requirement to use `yaml.FullLoader` ?

Reply:
The user is the one providing the data which means they should know if the data is trusted or not. 
If there's no loss in using safer ways we should go for it.

Reply:
FullLoader allows to use advanced (and unsafe) features of yaml such as:
```python
>>> someval = "123"                                     
>>> yaml.load("!!python/name:__main__.someval", Loader=yaml.FullLoader)
'123'
```
I think it's quite unusual to use these features that don't exist in JSON, so I'll replace with `safe_load` if it's OK.

Reply:
added use of `safe_load`